### PR TITLE
added payout docs

### PR DESCRIPTION
**Pull request by voith**

### CODE REVIEW

File: docs/source/payout.rst [hunk 1]
```diff
@@ -0,0 +1,51 @@
+Distribute Dividends
+====================
+
+The command line tool supports a simple CSV input to make payouts to shareholders.
+
+* Shares are exchanged here for ether or tokens like Gemini USD.
```
Comment (villesundell) [inline docs/source/payout.rst]: Shares itself are not exchanged/given (away). If you own security tokens, you are entitled to the payouts.

File: docs/source/payout.rst [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+* ``payout-distribute`` takes in the input csv with investor info.
+
+* ``create-holders-payout-csv`` is an optional command that can be used to generate the CSV. This CSV contains investor with
+    their respective shares. This basically fetches data by running ``token-scan`` in the background.
```
Comment (villesundell) [inline docs/source/payout.rst:31]: Do we need a wrapper? Could we improve `token-scan` by including CSV support?

File: docs/source/payout.rst [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+* Edit this csv to remove some holders or to modify the amount.
+
+* payouts can be made either in ether or any other payable ERC20 compliant token. use `--payment-type` flag in
+  ``payout-distribute`` to specify the type of payout. ``payment-type`` can either be `ether` or `token`. If ``payment-type``
+  is `token` then ``payment-token-address`` and ``payment-token-contract-name`` are required parameters. ``payment-token-address``
+  refers to the address of the deployed payout-token and  ``payment-token-contract-name`` refers to the name of the smart contract.
```
Comment (villesundell) [inline docs/source/payout.rst]: Name (and/or symbol) could be asked from the token itself?

File: docs/source/payout.rst [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+Example for making payouts:
+
+.. code-block:: shell
+
+    sto --config=myconfig.ini payout-distribute --csv-input="/Home/data/payout.csv" --payment-type="ether"
+
+
+This should output:
+
+
```
Comment (villesundell) [inline docs/source/payout.rst:81]: Also, we were planning that address(0) would be intentionally skipped, so the total amount of tokens (and their proportional value) would not change.

File: docs/source/payout.rst (hunk 2 above)
Comment (voith) [reply docs/source/payout.rst:31]: These two commands have different purpose. I don't want to combine the two into one(separation of concern).

**Comment by voith** [comment]
closing all stale PR's

**Pull request closed without merging**